rosa_install_access_delete_clusters/terraform/rosa-sts-creating-a-cluster-with-customizations-terraform.adoc

```diff
@@ -0,0 +1,68 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="rosa-sts-creating-a-cluster-with-customizations-terraform"]
+= Customizing a ROSA cluster with Terraform
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-sts-creating-a-cluster-with-customizations-terraform
+
+toc::[]
+
+You can create a {product-title} (ROSA) cluster by using Terraform to customize the cluster's details.
+
+This guide assumes an understanding of ROSA. See the additional resources for conceptual information about {product-title} clusters.
+
+.Prerequisites
+
+* You have generated and stored an link:https://console.redhat.com/openshift/token[offline {cluster-manager-first} token].
+* You have an AWS account with the correct permissions.
+* You have installed the `terraform` CLI tool.
+
+[discrete]
+[id="further_reading_{context}"]
+=== Further reading
+
+* xref:../../rosa_planning/rosa-cloud-expert-prereq-checklist.adoc[Prerequisites checklist for deploying ROSA using STS]
+* xref:../../rosa_planning/rosa-understanding-terraform.adoc[Preparing Terraform to install ROSA clusters]
+
+[id="rosa-sts-creating-a-cluster-with-customizations-terraform-creating"]
+== Creating a {product-title} cluster using Terraform
+The following Terraform cluster example shows how to create your account-wide IAM roles and a ROSA cluster with a managed OIDC configuration.
```

Comment:
```suggestion
The cluster creation process outlined below shows how to use Terraform to create your account-wide IAM roles and a default ROSA cluster with a managed OIDC configuration.

[NOTE]
====
Clusters are customized by editing the default `main.tf`, `variables.tf`, and `terraform.tfvar` files. Cluster customizations must be performed *before* cluster creation. To customize, copy the default Terraform file from the procedure below and make the desired changes.
==== 
```
* I know RH style doesn't like "desired". But I really think it fits here better than "necessary" or "required". But we could use either of those too.